### PR TITLE
network-ipfinder: add module

### PR DESCRIPTION
**Pull request by muntjaber**
Shows your public IP address (and country of origin) retrieved from `ipinfo.io`.

### CODE REVIEW

File: polybar-scripts/network-ipfinder/ipfinder.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (x70b1) [inline polybar-scripts/network-ipfinder/ipfinder.sh:1]: Use `#!/bin/sh` or describe why you need a bash.

File: polybar-scripts/network-ipfinder/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Script: Ipfinder
+
+Shows your public IP address (and country of origin) retrieved from `ipinfo.io`.
+
+![](screenshots/connected_vpn_up.png)
+
+## Dependencies
+
+```bash
+sudo apt install -y curl jq
```
Comment (x70b1) [inline polybar-scripts/network-ipfinder/README.md:10]: Use formatted list like on [notification-spacex](https://github.com/polybar/polybar-scripts/tree/master/polybar-scripts/notification-spacex).

File: polybar-scripts/network-ipfinder/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Script: Ipfinder
+
+Shows your public IP address (and country of origin) retrieved from `ipinfo.io`.
+
+![](screenshots/connected_vpn_up.png)
```
Comment (x70b1) [inline polybar-scripts/network-ipfinder/README.md:5]: Rename the screenshot to `1.png`.

File: polybar-scripts/network-ipfinder/ipfinder.sh (hunk 1 above)
Comment (muntjaber) [reply polybar-scripts/network-ipfinder/ipfinder.sh]: I've removed the bash builtin `declare` command.

**Comment by x70b1** [comment]
I impoved the script a bit. 
You can check it in `network-ipinfo.io`.

**Pull request merged**